src/doc/gui_manual/Plots/PlotTypes/MoleculePlot.rst

```diff
@@ -0,0 +1,140 @@
+.. _molecule_plot_head:
+
+Molecule Plot
+~~~~~~~~~~~~~
+
+The **Molecule** plot takes as input data with atoms and bonds (stored 
+internally as Vertices and Lines in a VTK PolyData structure) and renders it 
+as spheres and lines/cylinders
+
+.. figure:: ../images/Molecule_crotamine_species.png
+  :width: 45%
+
+  Molecule plot of crotamine, colored by element type, atoms shown with covalent radius, and no bonds.
+
+.. figure:: ../images/Molecule_crotamine_residue.png
+  :width: 45%
+
+  Molecule plot of crotamine, colored by residue type, atoms proportional to covalent radius emulating the CPK syle, bonds colored with adjacent atom color.
+
+.. figure:: ../images/Molecule_crotamine_scalar.png
+  :width: 45%
+
+  Molecule plot of crotamine, colored by a scalar quantity, no atoms shown, and bonds drawn as cylinders. 
+
+.. figure:: ../images/Molecule_crotamine_backbone.png
+  :width: 45%
+
+  Molecule plot of crotamine, colored by *backbone*, atom at same width as thicker cylinder-shaped bonds.
+
+
+
+Controlling how atoms are drawn
+"""""""""""""""""""""""""""""""
+
+.. _molecule_atoms:
+
+.. figure:: ../images/molecule_atoms.png
+
+   Molecule plot window Atoms tab
+
+
+The value *Spheres* for **Draw atoms as** means to draw spheres using 3D 
+geometry.  *Sphere Impostors* means to draw them using a single flat polygon 
+with an image of a sphere -- this requires support from graphics hardware and 
+can introduce some minor graphical artifacts, but is very fast. The value 
```

Comment:
Small grammar mistakes:
1. "...minor graphical artifacts, but is very fast." This can be left as is if the comma is removed, or it can be changed to "... minor graphical artifacts, but it is very fast." with the comma remaining.
2. "...you are only interested in seeing the bonds, and would like the atoms..." This is a similar situation. The sentence can remain the same if the comma is removed, or the comma can remain if it's changed to "...you are only interested in seeing the bonds, and you would like the atoms..."

Reply:
Thanks, done.